procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -22,13 +22,17 @@
     "import time\n",
```

Comment:
<p>Please add a comment somewhere that tells people to make sure they check rubinTV to make sure the images are appearing.</p><p>That's the canary in the coal mine :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>